app/controllers/nonresident/CalculatorController.scala

```diff
@@ -130,16 +130,16 @@ trait CalculatorController extends BaseController {
     }
   }
 
-  def calculateTotalGain(disposalValue: Double,
-                         disposalCosts: Double,
-                         acquisitionValue: Double,
-                         acquisitionCosts: Double,
-                         improvements: Double,
-                         rebasedValue: Option[Double],
-                         rebasedCosts: Double,
-                         disposalDate: Option[DateTime],
-                         acquisitionDate: Option[DateTime],
-                         improvementsAfterTaxStarted: Double): Action[AnyContent] = Action.async { implicit request =>
+  def calculateTotalGainModel(disposalValue: Double,
```

Comment:
would a name like buildTotalGainsModel be more descriptive of what this method actually does?